src/core/crypto.c

```diff
@@ -21,6 +21,7 @@
 */
 #include "crypto.h"
 #include <string.h>
+#include <stdbool.h>
```

Comment:
Do we use bools anywhere else or should we just use a uint8_t and check true / false against pound def?

Reply:
Resolved